src/go/rpk/pkg/cli/redpanda/admin/brokers/list.go

```diff
@@ -26,22 +50,25 @@ func newListCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 			bs, err := cl.Brokers(cmd.Context())
 			out.MaybeDie(err, "unable to request brokers: %v", err)
 
-			headers := []string{"Node-ID", "Num-Cores", "Membership-Status"}
+			headers := []string{"ID", "Host", "Port", "Rack", "Cores", "Membership", "Is-Alive", "Version"}
 
 			args := func(b *rpadmin.Broker) []interface{} {
-				ret := []interface{}{b.NodeID, b.NumCores, b.MembershipStatus}
+				ret := []interface{}{b.NodeID, b.InternalRPCAddress, b.InternalRPCPort, b.Rack, b.NumCores, b.MembershipStatus, *b.IsAlive, extractVersion(b.Version)}
```

Comment:
If rack is empty, I suggest replacing with `-`, so that the output is consistent for awk.